config/_default/menus.en.toml

```
@@ -19,6 +19,11 @@ weight = 5
   weight = 1
 
   [[main]]
+  name = "Vision"
+  parent = "Platform"
+  url = "/vision"
+  weight = 3
+
```

```suggestion
[[main]]
```

Menu file is invalid since the array indicator is missing for the next line.